docs/content/dagster-cloud/managing-deployments/alerts.mdx

```diff
@@ -102,6 +102,31 @@ Alert policies are configured on a **per-deployment basis**. This means, for exa
         <a href="/dagster-cloud/deployment/hybrid">Hybrid deployments</a> only.
       </td>
     </tr>
+    <tr>
+      <td>
+        <strong>Insights metric usage alert</strong>
+      </td>
+      <td>
+        <Note>A Dagster Cloud Pro plan is required to use this feature.</Note>
```

Comment:
I think it's fine to put this as the first sentence in the paragraph and bold it over using a callout. Looks a lil odd in the table.

Ex: `**This feature requires a Dagster Cloud Pro plan.**`